alf/algorithms/actor_critic_algorithm.py

```diff
@@ -47,18 +48,21 @@ def __init__(self,
         """Create an ActorCriticAlgorithm.
 
         Args:
+            observation_spec (nested TensorSpec): representing the observations.
             action_spec (nested BoundedTensorSpec): representing the actions.
-            actor_network (DistributionNetwork): A network that returns nested
+            env ():
+            config ():
```

Comment:
more specific comments for these two new args? If the base algorithm has the comments then probably just copy them here.

Reply:
comments added